### DISCUSSION TITLE
There was a problem with the server hosted by Amazon

### DISCUSSION THREAD

**Discussion by BBBIMAX** [LAPKB/Pmetrics#62] [Q&A] [answered]
Hello!
I'm a medical graduate student from China. I've been trying to use pmetrics software package recently, but I have some problems. After the software package is successfully added in rstudio, it cannot be compiled through "pmbuild()", which shows that the server cannot be connected. Through the test, the IP address of the server shows "timeout" for many times, and the details have been attached in the screenshot. 
![image](https://user-images.githubusercontent.com/34941699/168429255-7b7f99b1-581b-4c30-8015-dcbe80b59a56.png)
![image](https://user-images.githubusercontent.com/34941699/168429261-66ee7011-23a3-48af-b819-d8d342181d3e.png)

**Comment by Siel** [comment, marked as answer]
Hello @BBBIMAX,
That IP belongs to our old server, if you download the latest version of the code you will notice that that IP changed. https://github.com/LAPKB/Pmetrics/blob/c5c37312b2d911d9b0c704130a2ece34dc8a9a44/R/PMremote.R#L201

If the problem persists, you can skip the registration by using PMbuild(skipRegistration=T).

**Reply by BBBIMAX** [reply]
Thanks for very much. I well try the latest version！
